packages/handlers/openapi/src/index.ts

```diff
@@ -1,24 +1,67 @@
-import { readFileOrUrlWithCache } from '@graphql-mesh/utils';
-import { createGraphQLSchema } from 'openapi-to-graphql';
-import { Oas3 } from 'openapi-to-graphql/lib/types/oas3';
+import {
+  readFileOrUrlWithCache,
+  ResolverData,
+  parseInterpolationStrings,
+  stringInterpolator,
+} from '@graphql-mesh/utils';
+import { createGraphQLSchema } from '@ardatan/openapi-to-graphql';
+import { Oas3 } from '@ardatan/openapi-to-graphql/lib/types/oas3';
 import { MeshHandlerLibrary, YamlConfig } from '@graphql-mesh/types';
+import { fetchache, Request } from 'fetchache';
 
 const handler: MeshHandlerLibrary<YamlConfig.OpenapiHandler> = {
   async getMeshSource({ config, cache }) {
     const path = config.source;
-    const spec: Oas3 = await readFileOrUrlWithCache(path, cache, {
+    const spec = await readFileOrUrlWithCache<Oas3>(path, cache, {
       headers: config.schemaHeaders,
     });
 
+    const fetch: WindowOrWorkerGlobalScope['fetch'] = (...args) =>
+      fetchache(args[0] instanceof Request ? args[0] : new Request(...args), cache);
+
     const { schema } = await createGraphQLSchema(spec, {
+      fetch,
       baseUrl: config.baseUrl,
       headers: config.operationHeaders,
+      skipSchemaValidation: config.skipSchemaValidation,
```

Comment:
Please add all those (except `viewer` which needs a different fix, which I'll do later) to the config object. We should allow developers to override those.